### DISCUSSION TITLE
各タグのユーザーランキングの数字について

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#468] [要望・アイデア・不具合]
タイトルのとおりに各タグのユーザーランキングの数字について、見え方が変わっていることに気づきました。

恐らくですが、今までの「すべて」から「年間」の数字に変更されたということでしょうか？
Organizationランキングについては「すべて」がありますが、ユーザーランキングについても「すべて」があった方がよいと思います。

また、ホーム画面のユーザーランキングでは依然「すべて」が残っていますが、ホーム画面こそ「年間」の情報を見せた方が良いのではと思いました。

以上です。

**Comment by tanaka-asami415** [comment]
@Brutus03 

> 恐らくですが、今までの「すべて」から「年間」の数字に変更されたということでしょうか？

はい。先週変更いたしました。下記はその際のリリースノートになります🙇
https://qiita.com/release-notes/2023-04-19

> Organizationランキングについては「すべて」がありますが、ユーザーランキングについても「すべて」があった方がよいと思います。
また、ホーム画面のユーザーランキングでは依然「すべて」が残っていますが、ホーム画面こそ「年間」の情報を見せた方が良いのではと思いました。

ありがとうございます！検討いたします🙏

**Comment by ko1nksm** [comment]
私も各タグのユーザーランキングの期間「すべて」の復活を希望します。
頑張って上げてきた順位が無くなった時は悲しかったです。

流動性のある情報を表示するために「年間」を追加するのは良いと思いますが「すべて」を無くす必要はないはずです。
むしろ「年間」と「すべて」の両方を表示するほうが流動性のある情報かどうかを判断するのに役に立つはずです。
（つまり昔はコントリビューションが多かったが最近は少ないなと判断できる）

**Reply by tanaka-asami415** [reply]
@ko1nksm 
ご意見ありがとうございます。
悲しい思いをさせてしまい申し訳ありません🙇

パフォーマンス上の問題もあり、すぐに復活させるということが難しいため
時期はお約束できないのですが、実装方法を検討して、また「すべて」も表示させたいと考えております。
